src/placeos-driver/storage/redis-storage.cr

```diff
@@ -109,24 +109,24 @@ class PlaceOS::Driver::RedisStorage < PlaceOS::Driver::Storage
   @redis : Redis::Client? = nil
 
   def redis
-    @redis ||= self.class.new_redis_client
+    @redis ||= self.class.shared_redis_client
   end
```

Comment:
Since the mutex is internal, the client should be as well.
```suggestion
  private getter redis : Redis::Client { self.class.shared_redis_client }
```

Reply:
is it just me or is my suggested diff rendered really strangely?

Reply:
it does look like it merged then diffed or something